gradio/blocks.py

```diff
@@ -1204,15 +1204,20 @@ def queue(
             concurrency_count: Number of worker threads that will be processing requests from the queue concurrently. Increasing this number will increase the rate at which requests are processed, but will also increase the memory usage of the queue.
             status_update_rate: If "auto", Queue will send status estimations to all clients whenever a job is finished. Otherwise Queue will send status at regular intervals set by this parameter as the number of seconds.
             client_position_to_load_data: DEPRECATED. This parameter is deprecated and has no effect.
-            default_enabled: If True, all event listeners will use queueing by default.
+            default_enabled: Deprecated and has no effect. If True, all event listeners will use queueing by default.
```

Comment:
Just remove the second part, no? 
```suggestion
            default_enabled: Deprecated and has no effect.
```

Reply:
Yea lol 🙈